pkg/operator/controller/ingress/deployment.go

```diff
@@ -408,6 +409,27 @@ func desiredRouterDeployment(ci *operatorv1.IngressController, ingressController
 	}
 	env = append(env, corev1.EnvVar{Name: "SSL_MIN_VERSION", Value: minTLSVersion})
 
+	usingIPv4 := false
+	usingIPv6 := false
+	for _, clusterNetworkEntry := range networkConfig.Status.ClusterNetwork {
+		addr, _, err := net.ParseCIDR(clusterNetworkEntry.CIDR)
+		if err != nil {
+			continue
+		}
+		if addr.To4() != nil {
+			usingIPv4 = true
+		} else {
+			usingIPv6 = true
+		}
+	}
+	if usingIPv6 {
+		mode := "v4v6"
+		if !usingIPv4 {
+			mode = "v6_only"
```

Comment:
Didn't we do the following somewhere recently:

```go
mode = "v6"
```

Reply:
> ```go
> ```go
> mode = "v6"
> ```
> 
> 
>           
> ```

Oh, this is in the 2nd commit.

Reply:
Yeah, I kept it as two commits as it is on master and as openshift-cherrypick-robot would have done (I don't know how to ask openshift-cherrypick-robot to cherry-pick two PRs at once).  I can squash if that would be preferable.